doc/source/migrations/docs-deploy-pr-setup.rst

```diff
@@ -0,0 +1,153 @@
+.. _docs-deploy-pr-setup:
+
+Deploy documentation from a pull request
+========================================
+
+The ``ansys/action/doc-deploy-pr`` action handles both the deployment of html documentation from
+a pull request as well as it's automatic removal once the pull request is closed. Therefore,
+the ``closed`` github event should be added to ``on.pull_request.types`` to ensure that the action is
+triggered for documentation removal upon PR closure.
+
+To limit the number of deployed pull request documentation, a ``maximum-pr-doc-deployments`` input
+is provided, which is especially useful when a lot of pull requests can be opened at the same time
+in your repository or when the size of your documentation is quite large. In addition to this input,
```

Comment:
This sentence is long. Let's chop it so it is easier to read. Also, I bet Google documentation style states introduced the decision followed by the explanation:

```suggestion
The ``maximum-pr-doc-deployments`` input can be used to limit the number of documentation deployments. This is useful when multiple pull-requests exist in your repository.
```